odoo/addons/base/models/ir_ui_view.py

```diff
@@ -640,9 +640,10 @@ def handle_view_error(self, message, *, raise_exception=True, from_exception=Non
             when raising an exception, start with this traceback (default: start
             at exception creation)
         """
+        _t = _ if raise_exception else lambda eng: eng
         lines = [message]
         if self.name:
-            lines.append("\n%s: %s" % (_('View name'), self.name))
+            lines.append("\n%s: %s" % (_t('View name'), self.name))
```

Comment:
Is it correct? for example in ltr language does it make sense or should we fix this in something like 

`lines.append("\n%s" % _t('View name: %s', self.name))`